template.go

```diff
@@ -0,0 +1,26 @@
+package cloudconfig
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/k8scloudconfig/v_0_1_0_3500632"
```

Comment:
I thought about how to version this. There are certain limitations because of golang package name rules. For instance a package name has to start with a letter. The package name can be alphanumeric plus underscores. I want the package name and the import path to be equal. Here I tried to show a semver version, which in this case is `0.1.0`. I made this version up. The last part of the package name is the first 7 digits of the git commit hash. I am not sure we want or need this.

Reply:
How do you know what commit has it will be before committing?

Reply:
Right, the hash does not work that way. Good catch. I will just remove it.

Reply:
Done.